vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -0,0 +1,150 @@
+```
+You should see output similar to this:
+```shell
+Generating a RSA private key
+.....................................................................++++
+........................................................................................................................................................................................++++
+writing new private key to 'tls.key'
+-----
+```
+Now you can create a TLS Secret using the generated files.
+
+2. Go to `Workloads → Secrets → Create` and select `From YAML`. Use the following YAML as a starting point and then click `Create`.
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: entando-tls-secret
+data:
+  tls.key: ''
+  tls.crt: ''
+type: kubernetes.io/tls
+```
+3. Click on `Actions → Edit Secret` and use the ```Browse...``` buttons to upload the key and cert files.
+
+Now you'll create the `entando-ca-cert-secret` Secret, similar to what was done in Scenario 3 but this time using the self-signed certificate. 
+
+4. Go to `Workloads → Secrets → Create` and select `Key/value secret`
+5. Set the `Secret Name`, e.g. `entando-ca-cert-secret` 
+6. Set the `Key`, e.g. `tls.crt`
+7. Set the `Value` by clicking `Browse...` and loading the cert file from Step 1, e.g. `tls.crt`
+8. Click `Create`
+9. Next go to `Workloads → ConfigMaps` and update or create a ConfigMap named `entando-operator-config.` This is the ConfigMap used by the Operator to configure the deployments. You can [download the Entando Operator template](../devops/entando-operator.md#add-a-new-configmap) as a starting point. Set the "data/entando.ca.secret.name" and "data/entando.tls.secret.name" to match the names from above.
+```yaml
+data:
+ entando.ca.secret.name: entando-ca-cert-secret
+ entando.tls.secret.name: entando-tls-secret
+```
+10. Click `Create`
+
+Now let's create a new application, just like in Scenario 3 but with the self-signed SSL certificate.  
+
+11. Go to `EntandoApp` and click `Create instance`
+  - Keep the default `my-app` for your application name or select your own
+  - Select the EntandoApp version: `7.0`
+  - Set the `Ingress Host Name` as in Scenario 1 above. It should match the CN used to generate the cert in step 1.
+  - Keep the default `DBMS` as `postgresql`
+12. Click `Create`. The `Entando Operator` will now proceed to deploy the appropriate resources.
+
+Once the deployment is complete you can confirm that all routes use https with the self-signed certificate. You will still see security warnings in the browser.
+
+See the [Next Steps](#next-steps) below to continue your work with Entando.
+
+## Next Steps
+Once you've completed any of the scenarios above, you have several options.
+*  Check out `Networking → Routes` to see the URLs for the running services. Common starting points include the `Entando App Builder` (e.g. `http://entando.apps-crc.testing/app-builder/`) or the `Entando Application` itself (e.g. `http://entando.apps-crc.testing/entando-de-app/`).
```

Comment:
should Entando App Builder or Entando Application be backticked?

Reply:
Hmm, I thought we weren't in this case but now I've lost track. This tutorial wasn't consistent with it for Entando Operator either.